### ISSUE TITLE
The training settings and teacher models of training in DukeMTMC and MSMT17.

### ISSUE THREAD

**Issue by KiritoHugh** [NVlabs/DG-Net#44]
I used the same hyperparameters setting for DukeMTMC and MSMT17 as the setting in Market1501 training config file. And I used the pretrained Resnet model from https://kaiyangzhou.github.io/deep-person-reid/MODEL_ZOO as the teacher models. 

Although I got a pretty nice result in Market1501, the result in the other two datasets are really poor(even both poorer than their teacher models).

> Market1501:
torch.Size([3368, 1024])
Alpha:0.00 Rank@1:0.9374 Rank@5:0.9860 Rank@10:0.8490 mAP:0.8490
Alpha:0.50 Rank@1:0.9471 Rank@5:0.9872 Rank@10:0.8637 mAP:0.8637
Alpha:-1.00 Rank@1:0.9356 Rank@5:0.9837 Rank@10:0.8048 mAP:0.8048

> Duke:
torch.Size([2228, 1024])
Alpha:0.00 Rank@1:0.6683 Rank@5:0.7742 Rank@10:0.8043 mAP:0.4390
Alpha:0.50 Rank@1:0.6849 Rank@5:0.7873 Rank@10:0.8187 mAP:0.4652
Alpha:-1.00 Rank@1:0.6688 Rank@5:0.7733 Rank@10:0.8021 mAP:0.4291

> MSMT:
torch.Size([11659, 1024])
Alpha:0.00 Rank@1:0.4731 Rank@5:0.6029 Rank@10:0.6484 mAP:0.2736
Alpha:0.50 Rank@1:0.4754 Rank@5:0.6052 Rank@10:0.6544 mAP:0.2799
Alpha:-1.00 Rank@1:0.3923 Rank@5:0.5260 Rank@10:0.5779 mAP:0.1903

I also post the training loss log of DukeMTMC below.

- Does the teacher models' ability have a huge effect on the system's final ability?
- (小声哔哔：Could you share the hyperparameter training settings and teacher model on the other datasets?(Ｔ▽Ｔ)太穷了，跑一次要花50RMB，调不起参)
- I also wonder why you 大神们 know how to tune the hyperparameters, where I can learn some tricks about tuning? Only by practicing?

Your reply would be highly appreciated!

![DUKE](https://user-images.githubusercontent.com/26020754/82357466-c0fe6e00-9a37-11ea-970e-cab9c66b74a8.png)

**Comment by layumi** [comment]
Hi @KiritoHugh
Thanks for your attention on our paper.
This is my teacher config based on https://github.com/layumi/Person_reID_baseline_pytorch
DukeMTMC-reID
```
PCB: false
angle: false
balance: false
batchsize: 8
color_jitter: false
data_dir: ../DukeMTMC-reID/pytorch
droprate: 0.75
erasing_p: 0
fp16: false
gpu_ids: '0'
h: 256
lr: 0.01
name: Duke_batch8_lr0.010000_p0.00_d0.75
stride: 1
train_all: true
use_dense: false
w: 128
pool: avg
nclass: 702
```

MSMT-17
```
PCB: false
angle: false
balance: false
batchsize: 8
color_jitter: false
data_dir: ../MSMT17_V1/pytorch
droprate: 0.5
erasing_p: 0
fp16: false
gpu_ids: '0'
h: 256
lr: 0.01
name: MSMT_batch8_lr0.010000_p0.00_d0.50
stride: 1
train_all: true
use_dense: false
w: 128
pool: avg
nclass: 1041
```
